auth/config.go

```diff
@@ -31,6 +31,22 @@ type ConfigYellowfin struct {
 	SourceAccessFilter bool
 }
 
+// Permission holds all of the details to create the dynamic permission list
+type Permission struct {
+	ID          string `json:"id"`
```

Comment:
Isn't this thing actually an integer?

Reply:
The ID's in the permissions.ts in the @imqs/auth node module and the permissions.json file in this repo is saved as string values for some reason so I just stuck with the convention.